solidity/dashboard/src/components/Rewards.jsx

```diff
@@ -170,20 +167,39 @@ const RewardsComponent = ({ withdrawRewardAction }) => {
       : rewardsToReturn.slice(0, previewDataCount)
   }, [rewards, withdrawals, showAll, rewardFilter.status])
 
+  const amountTooltipText = (amount) => {
+    return `${displayEthAmount(amount, "gwei", null)} gwei`
+  }
+
   return (
     <>
       <Tile title="Total Balance" titleClassName="text-grey-70 h2">
         <header className="flex row center">
           {isFetching ? (
             <Skeleton className="h1 mb-1" styles={{ width: "25%" }} />
           ) : (
-            <>
-              <h1 className="balance">
-                {totalRewardsBalance}
-                <span className="h3 mr-1">&nbsp;ETH</span>
-              </h1>
-              <SpeechBubbleTooltip text="The total balance reflects the total Available and Acummulating rewards. Available rewards are ready to be withdrawn. Acummulating rewards become available after a signing group expires." />
-            </>
+            <div className="flex row mb-1 mt-1">
+              <TokenAmount
```

Comment:
We appear to be displaying a pointer cursor for these elements, but I'm not sure if clicking is a valid action? If it is, I'm not sure what it does, which is also not ideal :grin: If it isn't, then a pointer cursor doesn't seem appropriate---though I'm open to reasoning there.

Reply:
right, maybe `cursor: help`? But it's not a help heh 😆 so maybe we should set the `default`?

Reply:
I think `default` is just fine, yeah. I've seen `help` before, but it's always confusing tbh.